src/containers/Movie/MovieInfo/Actions/container.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+
+import {
+  getMovieAccountStates as getMovieAccountStatesAction,
+  toggleMovieWatchlist as toggleMovieWatchlistAction,
+  addMovieToFavorites as addMovieToFavoritesAction
+} from 'Store/movie/actions'
+import {
+  accountStatesLoadingSelector,
+  isOnWatchlistSelector,
+  isFavoriteSelector,
+  currentMovieIdSelector
+} from 'Store/movie/selectors'
+import ActionsComponent from './component'
+
+class Actions extends Component {
+  componentDidMount() {
+    const { getMovieAccountStates, movieId } = this.props
+    getMovieAccountStates(movieId)
+  }
+
+  handleToggleWatchlist = () => {
+    const { movieId, isOnWatchlist, toggleMovieWatchlist } = this.props
+    toggleMovieWatchlist(movieId, isOnWatchlist)
+  }
+
+  handleToggleBookmark = () => {
+    const { movieId, isFavorite, addMovieToFavorites } = this.props
+    addMovieToFavorites(movieId, isFavorite)
```

Comment:
can we get isOnWatchlist and isFavorite inside operation?

Reply:
@VladWithCookies  yes, but it needs more calculations. I already use this prop in component, so it will be simpler to pass it from the component into the action

Reply:
not sure about function signature. it is better to pass as small amount of arguments as possible

Reply:
done